### PR TITLE
[Tron-1] added startup menu (HI)

### LINKED ISSUE #13: Tron-1 Start-Up Menu
As a Player I want to have a startup menu so that  I can choose when to start playing 

**Acceptance Criteria:** 
Webiste has a startup menu which consist only of 'Play game' button

### PR DESCRIPTION
**Pull request by DGSR**
Implemented basic structure based on Minesweeper main menu
And added some styles

Closes #13

### CODE REVIEW

**Comment by rs1t** [comment]
Awesome design 👍

**Review by rs1t — CHANGES_REQUESTED** [review]
How about storing images in a separate folder?

**Comment by DGSR** [comment]
@rs1t Done as suggested

**Review by rs1t — APPROVED** [review]

**Review by DariaKapustina — APPROVED** [review]

File: Tron/gameField.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Minesweeper</title>
```
Comment (motya-not-dev) [inline Tron/gameField.html:5]: Change title, please

**Review by motya-not-dev — APPROVED** [review]

**Pull request merged**